src/ragger/backend/ledgercomm.py

```diff
@@ -69,6 +69,9 @@ def __exit__(self, *args, **kwargs):
         assert self._client is not None
         self._client.close()
 
+    def handle_usb_reset(self) -> None:
+        pass
```

Comment:
Maybe add a comment to use ledgerwallet here

Reply:
Eventually I implemented it on ledgercomm too. So for now both are still supporting the same features.